### PR TITLE
feat(CG-901): add application gateway support

### PR DESCRIPTION
**Pull request by zhouse51**
## Issue tracker links

_Add links to any relevant tasks/stories/bugs/pagerduty/etc_

*Example - dummy TODO project*

[TODO-123](https://autoclouddev.atlassian.net/browse/TODO-123)

## Changes/solution

_How does this change address the problem?_

## Testing

_Describe how the testing was done, plus evidence, if not covered by automated tests_

## Notes and considerations

_Add any additional notes and/or considerations_

## Dependencies

_Add dependencies on any other PRs, if applicable

### CODE REVIEW

File: src/services/applicationGateway/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,1273 @@
...
+    mutation: { add: false, delete: false }
+    subscription: false
+  ) {
+    id: String @search(by: [hash, regexp])
+    extendedLocation: azureApplicationGatewayExtendedLocation
+    loadBalancerFrontendIpConfigurations: [azureApplicationGatewayFrontendIPConfiguration]
+    ipConfigurations: [azureApplicationGatewayPrivateLinkServiceIpConfiguration]
+    networkInterfaces: [azureApplicationGatewayNetworkInterface]
+    provisioningState: String @search(by: [hash, regexp])
+    privateEndpointConnections: [azureApplicationGatewayApplicationGatewayPrivateEndpointConnection]
```
Comment (tyler-dunkel) [inline src/services/applicationGateway/schema.graphql:182]: @m-pizarro can you take a look at this? I imagine some of these names are too long to support the sql def

Comment (m-pizarro) [reply src/services/applicationGateway/schema.graphql:182]: sure, I'll take a look

Comment (m-pizarro) [reply src/services/applicationGateway/schema.graphql:182]: @tyler-dunkel I've been identifying the tables that need to be renamed but from what I see a lot have been added and most should be renamed. How about we go ahead with this merge and then I'll update my branch to fix all of them?

Comment (tyler-dunkel) [reply src/services/applicationGateway/schema.graphql:182]: sounds good to me!

**Pull request merged**